### ISSUE TITLE
Some assets (ADX in particular case) are not shown on wallet when they are received to multiaccount until manual adding

### ISSUE THREAD

**Issue by churik** [status-im/status-react#10786]
# Bug Report

## Problem
When receiving some assets to wallet, sometimes they are not shown until manual enabling them in assets (transaction is confirmed) and reopening app.
Reproducibility 100% with ADX.

#### Expected behavior
ADX amount is shown

#### Actual behavior
ADX is not shown, amount is displayed only after (1) enabling ADX manually in assets and (2) restarting app
Video is [here]( https://drive.google.com/file/d/1z8xgWZB3h1ct6kB5JSVlTXltH4h8l921/view?usp=sharing
)

### Reproduction

[comment]: # (Describe how we can replicate the bug step by step.)

- Open Status
- Create empty multiaccount A
- Send some ADX from another multiaccount to multiaccount A
- Multiaccount A: wait until transaction is confirmed 

### Additional Information

- Status version: release 1.4
- Operating System: Android, iOS

#### Logs
[status_logs-4.zip](https://github.com/status-im/status-react/files/4751598/status_logs-4.zip)

**Comment by churik** [comment]
Also confirmed with USDC

**Comment by hesterbruikman** [comment]
Wondering if this is a bug or designed behavior. @churik do you know if other assets are added upon being received? 

If I recall correctly we only show SNT and Ether by default and don't do path scanning on importing keys (or on other occasions) to see if you have other assets. Don't know if there's a difference for the situation of receiving assets to 'existing' accounts.

cc @johnlea-quiup

**Comment by churik** [comment]
@hesterbruikman 
it is an issue for sure, tried with AST and BLT - and they are shown after 3d confirmation on wallet without enabling them manually
![photo_2020-06-16 16 21 30](https://user-images.githubusercontent.com/4557972/84786768-739cfe80-afed-11ea-85df-5ec416019ab9.jpeg)

**Comment by churik** [comment]
Confirmed with AMB

**Comment by hesterbruikman** [comment]
Sounds like two problems, we might address separately

1. having to restart after manually enabling certain assets
2. having to add assets manually in the first place

I'd treat this GH issue as a focus on 1. For the second, I'll create a new issue in the `Import accounts` epic to enable scanning for assets on accounts. Once that's introduced in onboarding, I think it'll be more straightforward to add a way to call this method from within the wallet

**Comment by hesterbruikman** [comment]
From the video, it seems like the app could check if a received token is already checked in `Managed assets`, if not, suggest the user to add it. This depends on whether adding the asset requires connecting to collect token address and icon. If no IP address is exposed when collecting this information it could potentially be done automatically on behalf of the user

**Comment by hesterbruikman** [comment]
User report
> The wallet I'm watching from the status app doesn't show my balance of LRC. I've checked the box in manage assets multiple times even after phone restarts. I'm being told it must be a bug so I'm just giving you guys a heads up. Thanks.

Confirmed by another user

#### Reproducing in 1.5
Android 10 Samsung Galaxy s10e
LRC appears instantly after adding through manage assets

@churik Any recommendation on where to get ADX? Uniswap and Kyber don't seem to offer. 1inch does, but it keeps reloading i Status (will add [here](https://docs.google.com/spreadsheets/d/1qdtAjuIVIsTtgYmZ8G6HrBDjlRbAL2tdw7Xn7WtmaQE/edit#gid=0&range=A44))

**Comment by churik** [comment]
@hesterbruikman I can send you some, give me your address in dm

**Comment by churik** [comment]
Couldn't reproduce with AMB, ADX on Android 10 and Android 9 - issue is not reproducible anymore on 1.5

**Comment by Serhy** [comment]
Reopening this issue because it reproduced with UNI in my case of Android 8.1.

On AccountB - no UNI selected in Wallet.
AccountA sent to AccountB '10 UNI'. Result: no token appeared in main Wallet screen.
AccountB enables UNI from `Assets` view. It shows 0.
AccountB exchanges 5 UNI into ETH in Uniswap and correct amount is picked up in DApp.
AccountB after exchange successfully made and ETH amount updated in Wallet -> UNI amount keep with '0'
AccountB re-opens Status and UNI reflects correctly 5 tokens.

**Comment by churik** [comment]
Closed with "Scan tokens" button in wallet
